smartsim/_core/control/previewrenderer.py

```diff
@@ -74,14 +74,15 @@ def render(
     :type output_format: _OutputFormatString
     """
 
-    verbosity_level = _check_verbosity_level(verbosity_level)
+    verbosity_level = _check_verbosity_level(Verbosity(verbosity_level))
 
     loader = jinja2.PackageLoader("templates")
     env = jinja2.Environment(loader=loader, autoescape=True)
 
     env.filters["as_toggle"] = as_toggle
     env.filters["get_ifname"] = get_ifname
     env.filters["get_dbtype"] = get_dbtype
+    env.globals["Verbosity"] = Verbosity
```

Comment:
Are you sure you want to set this value to the _class_ Verbosity and not the parameter `verbosity_level`? I think this may not hold the value you want.

100% nitpick... can we make the global env name follow the same snake-case naming standard as `as_toggle`, `get_ifname`, and `get_dbtype`?

Reply:
I take it back. I had to look @ the template to learn what was happening. I am wrong on both accounts here.